src/app/component/Header/Header.component.js

```diff
@@ -381,7 +391,12 @@ export default class Header extends NavigationAbstract {
                   mods={ { isVisible, type: 'minicart' } }
                 >
                     <button
-                      onClick={ onMinicartButtonClick }
+                      onClick={ () => {
+                          onMinicartButtonClick();
+                          if (name === CART) {
+                              onMinicartOutsideClick();
```

Comment:
Just make sure you really want to run: `onMinicartButtonClick();` and `onMinicartOutsideClick();` together if `name === CART`. Just pay attention, will it work with only one function called if `name === CART`?